### PR TITLE
Add PostCSS, autoprefixer, and update dependencies

### PR DESCRIPTION
**Pull request by moelders**
- Added PostCSS and autoprefixer to webpack-less and webpack-sass packages.
- Updated webpack and webpack loaders dependencies.
- Updated version of example dependencies.
- Replaced example installation scripts with Lerna setup.

### CODE REVIEW

File: examples/less-extract/package.json [hunk 1]
```diff
@@ -2,13 +2,20 @@
   "name": "less-extract",
   "private": true,
   "scripts": {
-    "build": "cross-env NODE_ENV=production webpack",
+    "build": "npm run build:dev && npm run build:pro",
```
Comment (RecuencoJones) [inline examples/less-extract/package.json:5]: I think we should just build once for prod

File: .travis.yml [hunk 2]
```diff
@@ -8,7 +8,7 @@ install:
 
 script:
 - npm run lint
-- npm run lerna:bootstrap
+- npm run setup
```
Comment (RecuencoJones) [inline .travis.yml:11]: Should we use `setup:ci` here?

File: packages/webpack-less/extract.js [hunk 3]
```diff
@@ -3,8 +3,7 @@ const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const isProduction = process.env.NODE_ENV === 'production';
 
 const plugin = new MiniCssExtractPlugin({
-  filename: isProduction ? '[name].min.css' : '[name].css',
-  disable: !isProduction
```
Comment (RecuencoJones) [inline packages/webpack-less/extract.js:7]: Is it not necessary to disable the plugin anymore?

**Review by RecuencoJones — COMMENTED** [review]
Some minor comments / questions

File: examples/less-extract/package.json (hunk 1 above)
Comment (moelders) [reply examples/less-extract/package.json:5]: This is an example package, and the LESS (and SCSS) transpiler works diferently in dev and prod.

File: .travis.yml (hunk 2 above)
Comment (moelders) [reply .travis.yml:11]: I was working on removing `--no-package-lock` from Lerna configuration, but I could not tested it correctly and I did not push the lock files. So, the `setup:ci` should be removed instead.

File: packages/webpack-less/extract.js (hunk 3 above)
Comment (moelders) [reply packages/webpack-less/extract.js:7]: The new version does not have `disable` flag. I have tested dev and pro, and it does not throw any error.

**Review by RecuencoJones — APPROVED** [review]
LGTM!

**Pull request merged**